app/services/notifications/remove_each.rb

```diff
@@ -0,0 +1,27 @@
+module Notifications
+  class RemoveEach
+    # notifiable_collection is an Array of Hashes
```

Comment:
See the other comment :) Why not just receiving "notifications" ?

Reply:
I also answered the above comment :)